pycbc/conversions.py

```diff
@@ -842,21 +842,20 @@ def optimal_ra_from_detector(detector_name, tc):
     """For a given detector and GPS time, return the optimal orientation
     (directly overhead of the detector) in right ascension.
 
-
     Parameters
     ----------
     detector_name : string
         The name of the detector, e.g., 'H1'.
     tc : float
         The GPS time of the coalescence of the signal in the `ref_frame`.
-
     Returns
     -------
     float :
         The declination of the signal, in radians.
     """
     return optimal_orientation_from_detector(detector_name, tc)[0]
 
+
```

Comment:
Please undo all whitespace changes to `conversions.py`.

Reply:
I go with 1 empty line before "Parameters" and "Returns" in docstrings, and no particular choice for what comes before `return` statements (depends on context). I guess my main point was to keep this PR short, but if you think this whitespace change makes things more consistent, go ahead.

Reply:
I suppose it is fine to remove the empty line on line 845, but the one on line 852 should stay.

Two spaces *after* `return` statements is also fine.